docs/pages/access-graph/getting-started-on-prem.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: Getting started with Teleport Access Graph (TAG)
+---
+
+<Notice type="note">
+Teleport Access Graph is provided by default to users of the Teleport Team plan.
+Visit the [signup page](https://goteleport.com/signup/) to begin your free&nbsp;trial.
+</Notice>
+
+## Prerequisites
+
+<Tabs>
+<TabItem
+  scope={["enterprise"]} label="Self-Hosted Enterprise">
+
+- A running Teleport Enterprise cluster v14.3.4 or later.
```

Comment:
```suggestion
- A running Teleport Enterprise cluster v14.3.4 or later.
- An updated `license.pem` with Teleport Access Graph enabled. 
```

Reply:
Thanks, I added this to the list (as "Teleport Policy", as that's the product we sell), and removed the standalone notice box.